src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js

```diff
@@ -165,4 +178,33 @@ export default class AircraftTypeDefinitionModel extends BaseModel {
         this.speed = null;
         this.capability = null;
     }
+
+    /**
+     * Build the string used for `#icaoWithWeightClass`
+     *
+     * @for AircraftTypeDefinitionModel
+     * @method _buildIcaoWithWeightClass
+     * @return {string}
+     * @private
+     */
+    _buildIcaoWithWeightClass() {
```

Comment:
I'm going to go with `_buildTypeForStripView`, which I think is a happy medium between the few options.